erpnext/patches/v8_0/merge_student_batch_and_student_group.py

```diff
@@ -8,9 +8,9 @@
 
 
 def execute():
-
 	# for converting student batch into student group
-	frappe.reload_doctype("Student Group")
+	for doctype in ["Student Group", "Student Group Student", "Student Group Instructor", "Student Attendance"]:
+		frappe.reload_doctype(doctype)
```

Comment:
Use `reload_doc` here. For some instances, it is loading for the first time, hence `reload_doctype` will not work.